plugins/samanage/komand_samanage/connection/connection.py

```diff
@@ -24,3 +24,6 @@ def connect(self, params):
         self.api = SamanageAPI(token, is_eu_customer, self.logger)
 
         self.logger.info("Connect: Connection successful")
+
+    def test(self):
+        self.api.list_incidents_check()
```

Comment:
Please add `return {"success": True}` if the connection test works fine. Otherwise, kindly please throw `ConnectionTestException` with a proper `cause`, `assistance`, `data` 😄 
```suggestion
    def test(self):
        try:
            self.api.list_incidents_check()
            return {"success": True}
        except Exception as error:
            raise ConnectionTestException(cause=error.cause, assistance=error.assistance, data=error.data)
```